andela/hc-millennium-falcon
# #156577931 Users should be able to write blog posts on the health checks site

**What does this PR do?**

Allows a logged in user to create, edit, read and delete a blog post

**Description of the task to be completed**

When a user logs into the system, it should provide them with a blog tab where they can be able to click and view the current blog posts.

<img width="1242" alt="screen shot 2018-05-18 at 11 42 20" src="https://user-images.githubusercontent.com/17172747/40225026-9d878922-5a90-11e8-9c81-dcd5fb9ad616.png">

The user should also be able to create a new post by clicking on the green Add post button

<img width="942" alt="screen shot 2018-05-18 at 11 48 13" src="https://user-images.githubusercontent.com/17172747/40225327-6573fe8e-5a91-11e8-932a-5db76e6a7f49.png">

The user should also be able to create a new category by clicking on the green Add Category button

<img width="1178" alt="screen shot 2018-05-18 at 11 51 53" src="https://user-images.githubusercontent.com/17172747/40225522-f25835fe-5a91-11e8-9cdd-1caf4c46639e.png">

The user should be able to edit a blog post that they created 

<img width="1119" alt="screen shot 2018-05-18 at 11 55 43" src="https://user-images.githubusercontent.com/17172747/40226158-a2e389c2-5a93-11e8-9031-3217e7c2bbb8.png">


**How should this be manually tested?**

Clicking on Add Post or Add Category Button from the blog tab should take you to a page for you to add the relevant Post or Category. After adding the Post or Category you should be able to see your newly created Post or Category.

**Any background context you want to provide** 

No blog tab or application existed previously

**What are the relevant pivotal tracker stories?**

[#156577931](https://www.pivotaltracker.com/story/show/156577931)
